### ISSUE TITLE
Strange looking output when mixing changelog flags in one section

### ISSUE THREAD

**Issue by crywolf** [crywolf/mkchlog#10]
When we mix `title-is-enough: true` with `inherit: all` the produced output looks strange. I believe it's working according to the original example, but I think we should change it somehow. Any idea? @Kixunil 

Example:
```
Don't ignore read errors

When reading yaml config, mkchlog would silently ignore read erros which
could lead to confusion or even broken data (if the error happened
exactly at line boundary). This change passes the error to the caller
instead.

changelog:
        section: bug_fixes
        inherit: all
```
```
Skip merge commits

Merge commits does not follow required format and would be rejected as invalid. So they are just filtered out by calling `git log` command with `--no-merges` option.

changelog:
    section: bug_fixes
    title-is-enough: true
```
```
Do not display sections without content

Section containing empty subsection were previously displayed. Enhanced tests in Template.

changelog:
    section: bug_fixes
    title-is-enough: true
```

Output:
```markdown
## Fixed bugs

### Don't ignore read errors

When reading yaml config, mkchlog would silently ignore read erros which could lead to confusion or even broken data (if the error happened exactly at line boundary). This change passes the error to the caller instead.

* Skip merge commits

* Do not display sections without content
```

**Comment by Kixunil** [comment]
Fully agree. Perhaps put changes without description at the beginning so it becomes:

```
## Fixed bugs

* Skip merge commits

* Do not display sections without content

### Don't ignore read errors

When reading yaml config, mkchlog would silently ignore read erros which could lead to confusion or even broken data (if the error happened exactly at line boundary). This change passes the error to the caller instead.
```

Or put them into `### Other changes` section:

```
## Fixed bugs

### Don't ignore read errors

When reading yaml config, mkchlog would silently ignore read erros which could lead to confusion or even broken data (if the error happened exactly at line boundary). This change passes the error to the caller instead.

### Other changes

* Skip merge commits

* Do not display sections without content
```

The first one seems slightly better to me.

**Comment by crywolf** [comment]
Implemented by 34e48079c09808c4f65b34b995a008806cbcc83f, so I am closing it.
